src/main/java/com/minecolonies/coremod/client/gui/WindowHutBuilder.java

```diff
@@ -103,4 +198,24 @@ public void onUpdate()
             window.findPaneOfTypeByID(LIST_RESOURCES, ScrollingList.class).refreshElementPanes();
         }
     }
+
+    /**
+     * On Button click transfert Items.
+     *
+     * @param button the clicked button.
+     */
+    private void transferItems(@NotNull final Button button)
+    {
+
+        @NotNull final Label idLabel = (Label) button.getParent().getChildren().get(RESOURCE_ID_POSITION);
+        final int index = Integer.parseInt(idLabel.getLabelText());
+        @NotNull final ItemStack itemStack = resources.get(index).getItemStack();
+        @NotNull final Label quantityLabel = (Label) button.getParent().getChildren().get(RESOURCE_QUANTITY_MISSING_POSITION);
+        final int quantity = Integer.parseInt(quantityLabel.getLabelText());
+        final String buttonLabel = button.getID();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused "buttonLabel" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1481)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "buttonLabel". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1854)